src/Conversation/DataClients/Serializers/IntentNormalizer.php

```diff
@@ -136,7 +143,11 @@ public function denormalize($data, string $type, string $format = null, array $c
             $intent->setExpectedAttributes($data[self::EXPECTED_ATTRIBUTES]);
         }
 
-        return $intent;
+        if (isset($data[self::MESSAGE_TEMPLATES])) {
+            $messageTemplates = $this->serializer->denormalize($data[self::MESSAGE_TEMPLATES], MessageTemplateCollection::class);
```

Comment:
```suggestion
            $messageTemplates = $this->serializer->denormalize($data[self::MESSAGE_TEMPLATES], MessageTemplateCollection::class, $format, $context);
```

Just makes it easier when we're updating serialisers if the context is always passed around.